1.8/administration/change-logo.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: page
+title:  "Change Logo"
+categories: [administration]
+---
+
+Follow this procedure if you want to change the logo at the top-left of your board.
+
+- Upload your logo to your webserver.
+- Go to **Admin CP --> Themes --> Modify / Delete --> Edit your Theme**
```

Comment:
This should read as:
Go to ** Admin CP --> Themes --> Click on your theme name -->